### PR TITLE
fozzie-components@2.13.0 - Added Allure to Fozzie Components #globalconfig

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
Added Allure reporting for Webdriver and Jest.

### CODE REVIEW

**Review by siggerzz — DISMISSED** [review]

**Comment by JoshuaNg2332** [comment]
I have not updated changelogs for other packages due to this being minor updates and adding allure reporting to each package.

File: services/f-globalisation/src/mixins/tests/.jest-test-results.json
Comment (minrwhite) [inline services/f-globalisation/src/mixins/tests/.jest-test-results.json]: This file should be excluded, surely?

File: packages/f-user-message/src/components/tests/UserMessage.test.js
Comment (minrwhite) [inline packages/f-user-message/src/components/tests/UserMessage.test.js]: It may be worth adding `allure` to the list of globals in `.eslintrc` as per [here](https://eslint.org/docs/user-guide/configuring#specifying-globals) to prevent having these eslint comments everywhere it's used.

File: services/f-globalisation/src/mixins/tests/.jest-test-results.json
Comment (siggerzz) [reply services/f-globalisation/src/mixins/tests/.jest-test-results.json]: This file can be deleted as the plugin that required it has now been removed :)

Comment (JoshuaNg2332) [reply services/f-globalisation/src/mixins/tests/.jest-test-results.json]: I've removed this file now.

File: packages/f-user-message/src/components/tests/UserMessage.test.js
Comment (JoshuaNg2332) [reply packages/f-user-message/src/components/tests/UserMessage.test.js]: This has now been addressed and I've added the `allure` global support into the `.eslintrc` file.
I've also removed all comments out of each file.

**Comment by minrwhite** [comment]
I'm happy with this now, but will hold off approving until fozziebot is happy, and the merge will be clean. Make sure you're either rebased against main, or merged latest main in. You can also use `#globalconfig` tag as described [here](https://github.com/justeat/fozzie-components/blob/master/CONTRIBUTING.md#useful-pr-tags) to prevent having to bump all package versions, though updating the CHANGELOGS would be polite, as described in the link.

**Review by LTurns — APPROVED** [review]
looks good!

**Review by minrwhite — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

**Pull request merged**
